week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
-}
-
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+};
 
+function introduceMe(name, age) {
+return `Hello, my name is ${name} and I am ${age} years old`;
```

Comment:
Great. To format this nicely, you can indent the code inside the function using tab. By doing so, we can see where the content of the function is located clearly.